### DISCUSSION TITLE
any guide on how to enable and use gpio-sim?

### DISCUSSION THREAD

**Discussion by sombriks** [brgl/libgpiod#57] [Q&A] [answered]
i intend to run some CI for my wrapper but i have no clue on how could i provision a github runner to run gpio-sim.

anyone  tried that before? any hint is welcome.

**Comment by brgl** [comment, marked as answer]
I'm not well-versed in github runners but you'd basically need to be able to set the kernel config. You'd need to enable the `CONFIG_GPIO_SIM=m` option in kconfig.

**Reply by sombriks** [reply]
got it, thanks for the guidance!

the runner itself must offer a kernel able to run the simulator.
